server/controllers/admin.py

```diff
@@ -117,13 +119,17 @@ def assignment(cid, aid):
     # Convert TZ to Pacific
     assgn.due_date = convert_to_pacific(assgn.due_date)
     assgn.lock_date = convert_to_pacific(assgn.lock_date)
```

Comment:
Any ideas here? I tried thinking of a few solutions but none of them seemed right. The best one I have so far is a custom DateTimeField renderer (in _formhelpers) that calls `utils.local_time`

I'm planning on leaving this for later.